src/odo.ts

```diff
@@ -266,17 +275,19 @@ export class OpenShiftComponent extends OpenShiftObjectImpl {
         super(parent, name, contextValue, '', Collapsed, contextPath, compType, builderImage);
     }
     get iconPath(): Uri {
-        if (this.contextValue === ContextType.COMPONENT_PUSHED || this.contextValue === ContextType.COMPONENT || this.contextValue === ContextType.COMPONENT_NO_CONTEXT) {
-            let iconPath: Uri;
+        let iconPath: Uri;
+        if (this.locked) {
+            iconPath = Uri.file(path.join(__dirname, "../../images/component", 'progress.gif'));
```

Comment:
The gif images does not fit well with the design. We can use the label provided by vscode. `$(sync~spin)` to load the spinner.

Reply:
I would say it is perfect image for POC ;) created in 30 seconds.